### ISSUE TITLE
Model detection fails for `Ellipsis`

### ISSUE THREAD

**Issue by sasanjac** [drivendataorg/erdantic#124]
In `core.py`, only forms that don't have a `__qualname__` attribute are not considered models:
```python
def _add_if_model(self, model: type, recurse: bool) -> bool:
        """Private recursive method to add a model to the diagram."""
        try:
            key = str(FullyQualifiedName.from_object(model))
        except AttributeError as e:
            # May get typing special forms that don't have __qualname__ attribute
            # These are not going to be models
            if "__qualname__" in str(e):
                return False
            raise
```

However, e.g. when using ellipsis for type annotations, the error is as follows: `AttributeError: 'ellipsis' object has no attribute '__module__'. Did you mean: '__reduce__'?`

Maybe consider checking for more terms in the error message in order to catch non-modules?

**Comment by jayqi** [comment]
Hi @sasanjac, sorry for the slow response here.

I was able to reproduce your error with a model like this:

```
    class EllipsisModel(pydantic.BaseModel):
        ellipsis_field: Tuple[int, ...]
```

Can you confirm if this covers the kinds of annotations you're using with ellipsis, or do you have other kinds of examples?

**Comment by jayqi** [comment]
Hi @sasanjac @SebastianDD,

The fix for this has been released in v1.0.5. Please let me know whether or not this resolves your errors!

**Comment by SebastianDD** [comment]
> Hi @sasanjac @SebastianDD,
> 
> The fix for this has been released in v1.0.5. Please let me know whether or not this resolves your errors!

Thanks for the notice. We will check this. 

I have one addtional question, you may could answer: Asumming a pydantic class structure, is it possible to exclude classes? I know its possible to definde end-points via terminal_models, but in our case each class have an attribute of type "AttributeData", which we want to disable when drawing the diagram.
Thanks in advance.

**Comment by jayqi** [comment]
I made a separate issue to address the question about exclusions.
